pkgs/tools/backup/hpe-ltfs/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, fuse, icu, pkgconfig, libxml2, libuuid }:
+
+stdenv.mkDerivation rec {
+  version = "3.4.2_Z7550-02501";
+  pname = "hpe-ltfs";
+
+  src = fetchurl {
+    url = "https://github.com/nix-community/hpe-ltfs/archive/${version}.tar.gz";
+    sha256 = "0jwqpjlv92ld6v4ay3qjsjk94xq6gmk0kwdwz12bkaqaygymrcj7";
+  };
+
+  sourceRoot = "hpe-ltfs-3.4.2_Z7550-02501/ltfs";
+
+  buildInputs = [ 
+    fuse icu pkgconfig libxml2 libuuid
+  ];
+
+  meta = with stdenv.lib; {
+    description = "HPE LTO Filesystem (LTFS)-mount LTO tapes file filesystems";
+    homepage = https://support.hpe.com/hpesc/public/km/product/1009214665/Product;
+    license = licenses.gpl2;
+    maintainers = [ maintainers.redvers ];
+    platforms = platforms.linux;
```

Comment:
Can you add `downloadPage` to the github repo?

Reply:
As the original download location (I set that as homepage) or as the location in nix-community?

Reply:
is downloadPage valid inside of fetchFromGithub?

Reply:
```
nix-shell -I nixpkgs=/root/projects/nixpkgs -p hpe-ltfs
error: anonymous function at /root/projects/nixpkgs/pkgs/build-support/fetchurl/default.nix:38:1 called with unexpected argument 'downloadPage', at /root/projects/nixpkgs/pkgs/build-support/fetchzip/default.nix:17:2
(use '--show-trace' to show detailed location information)
```

Reply:
I meant could you add a downloadPage attribute to the `meta` that's the nix-community repo?

Reply:
resolved.